plugins/trace_api_plugin/include/eosio/trace_api/trace.hpp

```diff
@@ -89,6 +89,11 @@ namespace eosio { namespace trace_api {
       chain::packed_transaction_ptr       trx;
   };
 
+  struct block_trxs_entry {
+      std::set<chain::transaction_id_type> ids;
```

Comment:
Use `sha256_less` from `libraries/chain/include/eosio/chain/types.hpp` as the comparator of the `set`.

Reply:
Should I use `boost::container::flat_set`,  `fc::flat_set `or something else along with `chain:: sha256_less`?  `fc::raw::pack()` and `fc::raw::unpack()` would fail if using `std::set`.

Reply:
Used` fc::flat_set` .

Reply:
Changed back to `std::vector` because it has better performance on  new files lookup.